doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.rst

```diff
@@ -3,12 +3,88 @@ Release Notes
 #############
 
 New features, bug fixes, and improvements are regularly made to the SageMaker
-distributed model parallel library.
+distributed model parallelism library.
 
 
-SageMaker Distributed Model Parallel 1.14.0 Release Notes
+SageMaker Distributed Model Parallel 1.15.0 Release Notes
 =========================================================
 
+*Date: Apr. 25. 2023*
+
+**Currency Updates**
+
+* Added support for PyTorch v2.0.0.
+  However, the library does not support ``torch.compile`` at this release.
```

Comment:
LGTM.

Just a quick comment, not sure which one is better, e.g.
- However, the library does not support ``torch.compile`` at this release. vs
- Note that the library does not support ``torch.compile`` at this release yet.

Reply:
updated the sentence to incorporate with your second suggestion.